tests/unit/util/commands.js

```diff
@@ -0,0 +1,21 @@
+import { assertGameStateRow, assertGameState} from './helpers';
+
+
+export async function testConvertionGamestate(fixture){
+
+  const gameStateRow = await sails.helpers.gamestate.saveGamestate(fixture.gameState);
```

Comment:
Aren't these helpers synchronous? We shouldn't need async/await

Reply:
true, exception for the saveGamestate helper that create an entry in the database (create a GameStateRow )